### PR TITLE
#68411 Accessible Day Headers in Month View

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#68411**](http://central.tri.be/issues/68411)

**Code Review Note:** There are no "props" for this ticket.

### CODE REVIEW

**Comment by ggwicz** [comment]
Fixed some merge conflicts; this one's ready for another review! 🔍

File: readme.txt [hunk 1]
```diff
@@ -313,8 +313,10 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 * New - Made the "events" and "event" slugs translatable by WPML and other multilingual plugins [95026]
 * New - Introduced the `tribe_events_query_force_local_tz` filter to allow for forcing non-UTC event start and end times in Tribe__Events__Query [92948]
 * Fix - Allow The Events Calendar REST API to be disabled using the `tribe_events_rest_api_enabled` filter [97209]
-* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
 * Fix - Remove the errant `div.tribe-events-single-section` on the single event view when there is no venue [97615]
+* Tweak - Brought day number headers in the Month View into compliance with WCAG 2.0 accessibility standards when using the "Tribe Events" stylesheet [68411]
+* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659] 
+* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
```
Comment (bordoni) [inline readme.txt]: Are these entries relevant for this PR?

Comment (ggwicz) [reply readme.txt]: I was rearranging things in the readme, these aren't "new" with this PR just rearranged.

**Comment by ggwicz** [comment]
Fixed some merge conflicts; ready for another review 🔍

**Comment by ggwicz** [comment]
Fixed some merge conflicts again; this one's ready for another review 🔍

**Pull request merged**